lib/model/rofolder.go

```diff
@@ -16,113 +16,135 @@ import (
 
 type roFolder struct {
 	stateTracker
+	scan
```

Comment:
This is my aim in the long run, to have something called folder or similar. Which bundles all responsibility of a folder. (Try to implement writeOnly-folder type later on.)